BFH_USZ_PICC/BFH_USZ_PICC.iOS/BFH_USZ_PICC.iOS.csproj

```diff
@@ -26,7 +26,7 @@
     <MtouchArch>i386, x86_64</MtouchArch>
     <MtouchLink>None</MtouchLink>
     <MtouchDebug>True</MtouchDebug>
-    <CodesignKey>iPhone Developer: Florian Schnyder (6MQU9RCR8H)</CodesignKey>
+    <CodesignKey>iPhone Developer</CodesignKey>
```

Comment:
please exclude these changes. These are your debug Settings, not something for the generall App.